draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -882,6 +882,61 @@ The work on this draft was started at OAuth Security Workshop 2022 in Trondheim,
 
 TBD
 
+## Media Type Registration
+
+This section registers the "application/sdjwt-issuance" media type [@RFC2046] in
+the "Media Types" registry [@IANA.MediaTypes] in the manner described
+in [@RFC6838], which can be used to indicate that the content is a Combined Format for Issuance.
+This section also registers the "application/sdjwt-presentation" media type
+in the "Media Types" registry, which can be used to indicate that
+the content is a Combined Format for Presentation.
+
+### application/sdjwt-issuance
+
+*  Type name: application
+*  Subtype name: sdjwt-issuance
+*  Required parameters: n/a
+*  Optional parameters: n/a
+*  Encoding considerations: 8bit; application/sdjwt-issuance values are encoded as a series of base64url-encoded values, each separated from the next by a single tilde ('~') character.
+*  Security considerations: See the Security Considerations section of this document.
+*  Interoperability considerations: n/a
+*  Published specification: tbu
+*  Applications that use this media type: tbu
+*  Fragment identifier considerations: n/a
+*  Additional information:
+      Magic number(s): n/a
+      File extension(s): n/a
+      Macintosh file type code(s): n/a
+*  Person & email address to contact for further information: Daniel Fett, mail@danielfett.de
+*  Intended usage: COMMON
+*  Restrictions on usage: none
+*  Author: Daniel Fett, mail@danielfett.de
+*  Change Controller: IESG
+*  Provisional registration?  No
+
+### application/sdjwt-presentation
+
+*  Type name: application
+*  Subtype name: sdjwt-presentation
+*  Required parameters: n/a
+*  Optional parameters: n/a
+*  Encoding considerations: 8bit; application/sdjwt-presentation values are encoded as a series of base64url-encoded values (the last of which may be an empty string), each separated from the next by a single tilde ('~') character.
```

Comment:
dot character will be a separator too ...

Reply:
```suggestion
*  Encoding considerations: 8bit; application/sdjwt-presentation values are encoded as a series of base64url-encoded values (the last of which may be an empty string), each separated from the next by a single tilde ('~') character.
*  Encoding considerations: 8bit; application/sd-jwt-presentation values are encoded such that the first three base64url-encoded values are separated from the previous one by a single period ('.') character, and the following series of base64url-encoded values (the last of which may be an empty string or three base64url-encoded values separated from the next one by a single period ('.') character) are each separated from the previous one by a single tilde ('~') character.
```

Reply:
my best attempt to put it in one sentence

Reply:
```suggestion
*  Encoding considerations: 8bit; application/sdjwt-presentation values are encoded as a series of strings, each separated from the next by a single tilde ('~') character; the first and last strings consist of dot-separated base64url-encoded values, all other strings are base64url-encoded values; the last string may be empty.
```